run_dir/static/css/index.css

```diff
@@ -0,0 +1,17 @@
+#project_quotas .plot-column {
+	height: 60px;
+	padding: 0;
+}
+
+tr td:first-child {
+	width: 150px;
+}
+
+.update_table {
+	overflow: hidden;
+}
+
+.quotas_table {
```

Comment:
Is this meant to be `#project_quotas`? Also for the resizing bug thing it's probably best to set it just for the problem cells. So:
```css
#project_quotas tr td:nth-child(2){
    overflow: hidden;
}
```